.changeset/yellow-hairs-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/ea-bootstrap': minor
+---
+
+Removed UUID environment variable
```

Comment:
Could you add an explanation for how this impact users, and what users that relied on `UUID` should do now? (Basically, no action needed for EAs with the same name. Need to set `CACHE_KEY_GROUP` to split EAs with the same name).

Also wondering if this justifies a major version bump - since it can be a breaking change... @alejoberardino opinions?

Reply:
@boxhock Does this look better ? 
`Removed UUID environment variable. UUID was used for generating cache keys. If you were not using UUID environment variable and/or running single instance of EA there is no action needed. If you were using UUID to run multiple instances of the same EA with shared cache, you should use CACHE_KEY_GROUP environment variable instead. Applicable only in remote cache scenarios like when using redis for cache.`

As for major version bump i also thought about it. I don't know how common is the case where this change can break something.

Reply:
> If you were using UUID to run multiple instances of the same EA with shared cache, you should use CACHE_KEY_GROUP environment variable instead.

Technically they shouldn't have to set anything now if running multiple EAs.`CACHE_KEY_GROUP` should be used if they want to separate caches between multiple identical EAs using the same Redis cache.